libipl/libipl.C

```diff
@@ -359,3 +363,29 @@ void ipl_error_callback(const ipl_error_info& error)
 		return;
 	g_ipl_error_callback_fn(error);
 }
+
+void ipl_plat_error_callback(struct pdbg_target *target)
```

Comment:
Plat error callback is based on fapirc provided bty hwp , better to pass the fapirc value. to add callout based on fapirc value. existingflow is good from cfam/scom failure related handling. Also add deconfig required flag , by defailut make it required , only the procedures like prep- for poweroff path you can pass as deconfig not required.